database/seeds/MessageSettingTableSeeder.php

```diff
@@ -22,8 +22,7 @@ public function run()
             $setting->subject = $message['subject'];
             $setting->body = $message['body'];
             $setting->pro_tip = $message['pro_tip'];
-            $setting->signoff = $message['signoff'];
-            $setting->shoutout = $message['shoutout'];
+            $setting->signoff = isset($messagse['signoff']) ? $messagse['signoff'] : '';
```

Comment:
Just curious but why do we need to set this to an empty string if it's allowed to be `nullable()` via the definition in the migration? Plus we don't want to have empty strings in the DB.